docs/AI_CONTENT_GENERATION.md

```
@@ -0,0 +1,461 @@
+
+# 6. Commit and test
+git add packages/content-generator/generated/
+git commit -m "chore: Generate 12 races and 3 tech trees"
+```
+
+## Troubleshooting
+
+### Common Issues
+
+**API Key Error**:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)